package.yaml

```diff
@@ -52,15 +52,16 @@ when:
       # Introduced in GHC 8.
       - -Wredundant-constraints
       # Older Cabal generates warnings with its Paths_* module.
-      - -Wmissing-import-lists
```

Comment:
ignore all the changes in this file, `freer-effects` just has an annoying coding style, and i copied the yaml file from some other exploratory work i was doing